includes/Core/CLI/CLI_Command.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Site Kit CLI Command
+ *
+ * @package   Google\Site_Kit\Core\CLI
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\CLI;
+
+use Google\Site_Kit\Context;
+use WP_CLI_Command;
+
+/**
+ * Base CLI Command class.
+ *
+ * @since n.e.x.t
+ */
+class CLI_Command extends WP_CLI_Command {
+
+	/**
+	 * Plugin context.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @access protected
+	 * @var \Google\Site_Kit\Context
```

Comment:
No need for `@access` here, and `@var` for classes use the same name as would be used in the code. Since this has a `use` statement, it would just be `Context`.
```suggestion
	 * @var Context
```

Reply:
Updated.